### DISCUSSION TITLE
Bot Names Suggestions

### DISCUSSION THREAD

**Discussion by heiheihang** [LalaCuteGroup/lala-referral#15] [Ideas]
We will need a bot name and a TG ID for the Bot.
Leave all suggestions and we will make a poll

**Comment by heiheihang** [comment]
Lala-Referral-Bot
Lala_Referral_Bot

**Comment by jonowo** [comment]
Bot username can only consist of alphanumeric characters + underscore, and need to end with `bot` (case insensitive).
Bot display name can be any string.

ඞඞඞ

**Reply by GainFatKirby** [reply]
Then maybe `lala_referral_bot` for the bot username? It seems more consistent.

**Reply by slience-suzuka** [reply]
lala_referral_bot or lala_job_bot
